app/editor/src/features/content/list-view/ContentFilter.tsx

```diff
@@ -132,8 +132,8 @@ export const ContentFilter: React.FC<IContentFilterProps> = ({ search }) => {
                 name="isPrintContent"
                 label="Print Content"
                 tooltip="Newspaper content without audio/video"
-                value={filter.printContent}
-                checked={filter.printContent}
+                value={filter.contentType === ContentTypeName.PrintContent}
```

Comment:
This throws me off, why do we need to set both `value` and `checked`?  Wouldn't the value just be `ContentTypeName.PrintContent`?

Reply:
We don't technically need the value as we are just using if it is checked or not to determine the filter (and we are not submitting) - I could remove the value attribute from the rest of the checkboxes if that is desired! Just wasn't sure if there was a reason behind the scenes